packages/grid/_modules_/grid/utils/domUtils.ts

```diff
@@ -78,7 +78,7 @@ export function getGridCellElementFromIndexes(
   { colIndex, rowIndex }: GridCellIndexCoordinates,
 ) {
   return root.querySelector(
-    `:scope .${GRID_CELL_CSS_CLASS}[aria-colIndex='${colIndex}'][data-rowIndex='${rowIndex}']`,
+    `:scope .${GRID_CELL_CSS_CLASS}[aria-colIndex='${colIndex + 1}'][data-rowIndex='${rowIndex}']`,
```

Comment:
could we change the aria here as well pls?

Reply:
Actually this method `getGridCellElementFromIndexes` is not being used anywhere and I think we are exporting `domUtils.ts` in bundle. Is it for users? Should I remove the method itself?

Reply:
yes remove it please 👍

Reply:
Removed